### PR TITLE
[TASK] set binary as default for commands that accept the flag

### PR DESCRIPTION
**Pull request by boxbag**

### CODE REVIEW

**Comment by boxbag** [comment]
Hold off on merging, need to get coverage back up

File: src/js/ripple/remote.js [hunk 1]
```diff
@@ -1038,6 +1038,47 @@ Remote.prototype.requestLedgerCurrent = function(callback) {
...
+ * @param {LedgerData} ledgerData
+ *
+ * @return {Request} request
+ */
+
+Remote.prototype.requestLedgerData = function(options, callback) {
+  var request = new Request(this, 'ledger_data');
+
+  request.message.binary = options.binary !== false;
+  request.message.ledger_hash = options.ledger_hash;
```
Comment (wltsmrz) [inline src/js/ripple/remote.js]: This command also accepts `ledger_index`. Probably best to reuse the request.selectLedger heuristic

**Comment by boxbag** [comment]
Updated

**Comment by wltsmrz** [comment]
LGTM

**Comment by boxbag** [comment]
Hold off again, found another issue with binary transaction parsing

**Comment by boxbag** [comment]
Updated to handle parsing of partial payments for `tx` and `account_tx` commands

**Comment by boxbag** [comment]
Rest tests currently not passing, working on them currently

**Comment by wltsmrz** [comment]
LGTM

**Pull request merged**